src/api.rs

```diff
@@ -2239,3 +2275,22 @@ pub struct AssembleArtifactsResponse {
     pub missing_chunks: Vec<Digest>,
     pub detail: Option<String>,
 }
+
+#[derive(Debug, Serialize, Clone)]
+pub struct PatchSet {
+    pub path: String,
+    #[serde(rename = "type")]
+    pub ty: String,
```

Comment:
Why not just use `type` instead and skip the renaming?

Reply:
@kamilogorek I believe `type` is a reserved keyword. We do something similar here: `sentry-cli/src/commands/difutil_find.rs` line 29

Reply:
👌